src/ClusterBootstrap/deploy.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
-rm -rf deploy/!\(bin\) cloudinit* !\(config\).yaml
+shopt -s extglob
```

Comment:
What is this for?

Reply:
clean previous deployment, keep large-size universal binaries and config.yaml only.